src/config_constants.py

```diff
@@ -0,0 +1,8 @@
+import os
+import yaml
+
+config_path = os.path.join(os.path.dirname(__file__), "..", "config.yaml")
+with open(config_path, "r") as file:
+    config = yaml.safe_load(file)
+
+DENSE_UNITS = config["dense_units"]
```

Comment:
To jest takie sobie, bo trzeba wszystkie elementy do zmiennej pakować. Prawdopodobnie możemy to zmienić w PRce dotyczącej configów, więc zostawmy.